dell/xps/13-9320/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, pkgs, ... }: {
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+    ../../../common/pc/ssd
+    ./webcam.nix
```

Comment:
after these commits [1](https://github.com/NixOS/nixpkgs/pull/295508) and [2](https://github.com/NixOS/nixpkgs/pull/296164) , we probably can remove the webcam file and just add 

```
 hardware.ipu6 = {
    enable = true;
    platform = "ipu6ep";
  };
```